### PR TITLE
Update Textarea.php

### PR DESCRIPTION
**Pull request by dabba39**
escape value

### CODE REVIEW

**Comment by harikt** [comment]
I feel it is the users responsibility to do the same. 

Eg : sometimes we may not need to escape it . So 

``` php
echo $helper->input(array(
    'type'    => 'textarea',
    'name'    => $this->escape()->html($value),
    'value'   => 'bar',
    'attribs' => array()
));
```

One thing to note is when using `Aura.Input` users may need to write the escaping there.

**Comment by harikt** [comment]
Also we will not know whether the user need to escape html itself in textarea. May be it is for users to submit `css`, `js` ... ?

**Comment by dabba39** [comment]
i think it should be on by default, because a textarea is embedded in html. An input like '< script >alert('hello!')< /script >' should not popup an alert.
It should be just the same behaviour an < input type="text" /> tag i.e. has already

**Comment by pmjones** [comment]
@dabba39 is correct here. I can't believe I overlooked this. I am reviewing the rest of Aura.Html for similar oversights.

**Comment by harikt** [comment]
I don't think I agree with the change. Is there a way you can turn off the escaped value then?

There can be instances where you do want to get raw data.

If the data should be escaped it could be written on the Aura.Input and apply the Filters .

File: src/Helper/Input/Textarea.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ class Textarea extends AbstractInput
     public function __toString()
     {
         $attribs = $this->escaper->attr($this->attribs);
-        return "<textarea {$attribs}>{$this->value}</textarea>";
+        $value = $this->escaper->html($this->value);
+        return "<textarea {$attribs}>{$value}</textarea>" . PHP_EOL;
```
Comment (harikt) [inline src/Helper/Input/Textarea.php:33]: may be `PHP_EOL` also not needed.

**Pull request merged**